Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -1136,21 +1166,19 @@ void FitPeaks::calculateFittedPeaks() {
       if (chi2 > 10.e10)
         continue;
 
-      for (size_t ipar = 0; ipar < num_peakfunc_params; ++ipar) {
-        double value_i = m_fittedParamTable->cell<double>(row_index, 2 + ipar);
-        peak_function->setParameter(ipar, value_i);
-      }
+      // DEBUG CHECK  TODO Remove after testing are passed
```

Comment:
Comment says this should be removed. If it is inexpensive, leave it in and remove the comment.